single-line-diagram-color/src/main/resources/base-voltages.yml

```diff
@@ -0,0 +1,18 @@
+baseVoltages:
+    - name: "0"
+      minValue: 0
+      maxValue: 1
+      color: "#808080"
+      profile: "RTE"
+    - name: "400"
+      minValue: 300
+      maxValue: 500
+      color: "#FF3333"
+      profile: "RTE"
+    - name: "225"
+      minValue: 180
+      maxValue: 300
+      color: "#33FF33"
+      profile: "RTE"
+
+defaultProfile: "RTE"
```

Comment:
Could you rename it Default?

Reply:
Is it "RTE" that should be renamed to "Default", or the property name "defaultProfile" ?

Reply:
RTE